### PR TITLE
Rename files

### PR DESCRIPTION
**Pull request by duglin**
if #521 and #529 go in, then rename for consistency

Just look at the last commit to see the changes specific to this PR and not #521 and #529

### CODE REVIEW

**Comment by duglin** [comment]
CI errors are expected until merged.
Ready for review

**Comment by markpeek** [comment]
LGTM

File: extensions/partitioning.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ those events.
 The partitionkey attribute extension uses the key `partitionkey` for
 in-memory formats.
 
-### Transport format
+### Protocol format
```
Comment (deissnerk) [inline extensions/partitioning.md]: I'm not sure if *in-memory format* vs. *protocol format* are still the right categories. What about *stand-alone event format* and *protocol binding*? Or *structured-mode message* vs. *binary-mode message*?

File: amqp-protocol-binding.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ This document is a working draft.
 
 ## 1. Introduction
 
-[CloudEvents][ce] is a standardized and transport-neutral definition of the
+[CloudEvents][ce] is a standardized and protocol-neutral definition of the
```
[the remark this answers is not in the document]
Comment (duglin) [reply amqp-protocol-binding.md]: ok I got all of these in all of the files

File: avro-format.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ This document is a working draft.
 
 ## 1. Introduction
 
-[CloudEvents][ce] is a standardized and transport-neutral definition of the
+[CloudEvents][ce] is a standardized and protocol-neutral definition of the
```
[the remark this answers is not in the document]
Comment (duglin) [reply avro-format.md]: done

File: extensions/partitioning.md (hunk 1 above)
Comment (duglin) [reply extensions/partitioning.md]: This feels like more than a syntax fix so I'd prefer to leave that for a follow-on PR - if you want to open that one up

**Comment by duglin** [comment]
Got 2 LGTMs and since this is just syntax fixes I'm going to merge. If anyone thinks there's something I missed or that a change was made incorrectly please raise an issue and we'll address it immediately.

**Pull request merged**